temporian/implementation/pandas/operators/tests/sum/test_data/different_shape.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""PandasSumOperator - different shape test.
+
+Tests that data_1 and data_2 cannot have different shapes.
+"""
+import pandas as pd
+
+from temporian.implementation.pandas.data.event import PandasEvent
+
+INPUT_1 = PandasEvent(
+    {
+        "A": [
+            1,
+            2,
+        ],
+        "timestamp": [
+            pd.Timestamp("2020-11-09"),
+            pd.Timestamp("2020-11-10"),
+        ],
+        "B": [
+            3,
+            4,
+        ],
+    }
+).set_index(["timestamp"])
```

Comment:
I think this formatting can be improved - we don't need to split lists into multiple lines if they only have 2 elements.